tools/bin/mbedtls-kick-ci

```diff
@@ -0,0 +1,100 @@
+#!/bin/sh
+
+set -eu
+
+script=$(basename $0)
+usage="$script: usage: $script PR_NUMBER [all|internal|internal-pr-head|internal-pr-merge|openci|...]"
+
+host_internal_ci=jenkins-mbedtls.oss.arm.com
+host_open_ci=mbedtls.trustedfirmware.org
+
+# ~/bin/jmt needs to contain the Jenkins user and API token for each CI, e.g.
+#
+# JENKINS_INTERNAL_USER_ID=user-id-for-internal-ci
+# JENKINS_INTERNAL_API_TOKEN=1111111111111111111111111111111111
+# JENKINS_OPENCI_USER_ID=user-id-for-open-ci
+# JENKINS_OPENCI_API_TOKEN=2222222222222222222222222222222222
+#
+# To get the user ID, go to Jenkins, e.g. https://mbedtls.trustedfirmware.org/,
+# click your name in the top right, then the Status page should show your Jenkins
+# user ID. Both internal and OpenCI user names will almost certainly be the same
+# as your GitHub ID.
+#
+# To get an API token, choose Configure from your user page, click Add new Token,
+# give it a name, then click Generate. You get one chance to copy it, so do it
+# now! Then put it into ~/bin/jmt.
+#
+# Make sure ~/bin/jmt is chmod 0600.
+
+. ~/bin/jmt
```

Comment:
Please make this path configurable or make it a dot file.

Reply:
Done in https://github.com/Mbed-TLS/mbedtls-docs/pull/136/commits/8953ced81a670a01d4ddf5edb4c00a8e59903857, with the path now under `~/.config`. It can of course still be a symbolic link.